src/index.html

```diff
@@ -1,13 +1,409 @@
+                      src="../src/images/photos/details-4.jpg"
+                      alt="The phone with MyBike app"
+                      class="features__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--tablet-5-6 grid__item--desktop-9-12">
+                  <div class="features__photo-container">
+                    <img
+                      src="../src/images/photos/details-3.jpg"
+                      alt="Place the phone on a bike frame"
+                      class="features__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+              <div class="features__description">
+                <div class="grid grid--desktop">
+                  <div class="grid__item grid__item--tablet-1-6 grid__item--desktop-1-8">
+                    <h3 class="features__title">
+                      Range & Integrated lights
+                    </h3>
+                    <p class="features__text">
+                      The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="details__features features">
+            <div class="features__photos">
+              <div class="grid">
+                <div class="grid__item grid__item--tablet-1-2 grid__item--desktop-1-4">
+                  <div class="features__photo-container">
+                    <img
+                      src="../src/images/photos/details-6.jpg"
+                      alt="The phone with MyBike app"
+                      class="features__photo"
+                    >
+                  </div>
+                </div>
+                <div class="grid__item grid__item--tablet-3-6 grid__item--desktop-5-12">
+                  <div class="features__photo-container">
+                    <img
+                      src="../src/images/photos/details-5.jpg"
+                      alt="Place the phone on a bike frame"
+                      class="features__photo"
+                    >
+                  </div>
+                </div>
+              </div>
+              <div class="features__description">
+                <div class="grid grid--desktop">
+                  <div class="grid__item grid__item--tablet-1-6 grid__item--desktop-1-8">
+                    <h3 class="features__title">
+                      Hydraulic disc brakes & Lightweight
+                    </h3>
+                    <p class="features__text">
+                      Brakes with total stopping power the second you make contact. The removable battery has up to 70km battery autonomy and weighs only 2.4 kg. Lights integrated into the frame give you always-on visibility day and night.
+                    </p>
```

Comment:
move the text to a new line, try to keep your code without horizontal scroll

Reply:
That's good advice, I missed that. Thank you1